app.js

```diff
@@ -20,3 +20,265 @@ navLink.forEach((link) => link.addEventListener('click', () => {
+  },
+];
+
+/* pop up window */
+
+const popUpData = [
+  {
+    projectDescription: `Lorem ipsum dolor sit, amet consectetur adipisicing elit. Modidicta suscipit iusto, iste earum quod ad optio aliquid, voluptatum
+    aspernatur repellendus! Non quidem, id adipisci aut temporibus fuga dolor hic delectus numquam. Quo eligendi nesciunt reprehenderit voluptatem iure impedit tenetur nisi, officia,
+    numquam minima voluptatibus? Obcaecati maxime laudantium nihil beatae?`,
+    liveDemo: 'https://christianbiring1.github.io/My_Portfolio/',
+    sourceCode: 'https://github.com/christianbiring1/My_Portfolio',
+  },
+];
+/* Grab the container i.e work-section-container */
+
+const workSectionContainer = document.querySelector('.work-section');
+
+/*  Generate the work-section content of card 1 & 3  */
+
+projectInfo1.forEach((work) => {
+  const layout = document.createElement('article');
+  const layoutClass = ['card-work', 'card-work1'];
+  layout.classList.add(layoutClass[0]);
+  layout.classList.add(layoutClass[1]);
+  layout.innerHTML = `<figure><img scr= "${work.featureImage}" class="${work.classMobileImg}">
+    <img src= "${work.featureImageDesk}" class="${work.classDeskImg}">
+    <figcaption><h2>${work.caption}</h2></figcaption></figure>
+    <div class='aside'>
+      <div class='type'>
+       <h2>${work.projectName}</h2>
+       <ul>
+          <li class="title">${work.feature.title}</li>
+          <li class="domain">${work.feature.domain}</li>
+          <li class="year">${work.feature.year}</li>
+       </ul>
+      </div>
+      <p class="description">${work.projectDescription}</p>
+      <div class="languages languages-desk">
+        <ul>
+          <li><span class="html">${work.technologies[0]}</span></li>
+            <li><span class="css">${work.technologies[1]}</span></li>
+            <li><span class="js">${work.technologies[2]}</span></li>
+          </ul>
+      </div>
+      <button class="seeProject">See project <img src= "${work.featureImageDesk}" class="${work.classDeskImg}"></button>
+    </div>`;
+  workSectionContainer.appendChild(layout);
+
+  /* Generation of the popUp container */
+  /* Generate popUp window */
+
+  const popLayer = document.querySelector('.layer');
+  const popUpWindow = document.querySelector('.popup-window');
+  const sample = document.querySelectorAll(`.${work.classDeskImg}`);
+  sample.forEach((image) => {
+    image.addEventListener('click', () => {
+      work.featureImageDesk = image.getAttribute('src');
+      popUpWindow.innerHTML = `<h2>${work.projectName}</h2>
+      <div class="header">
+        <ul class="feature">
+          <li class="title">${work.feature.title}</li>
+          <li class="domain">${work.feature.domain}</li>
+          <li class="year">${work.feature.year}</li>
+        </ul>
+        <button class="close">&times;</button>
+      </div>
+      <img src="${image.getAttribute('src')}" class="card-work" alt="" />
```

Comment:
Thanks @prolajumokeoni  for the review [Optional] as you mention.
Basically, the approach I took was a little bit different from the template of the fact that if I had a container of the pop-up window It would be a little bit easier for me to make the exact same img as the one in Figma but as I forget since the beginning of implementing the popup modal it will be somehow difficult for me to make it look exactly like that. But anyway I have reduced the width of it as required.